src/indexes.rs

```diff
@@ -302,7 +274,7 @@ impl Index {
     /// # movie_index.delete().await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
     /// # });
     /// ```
-    pub async fn get_documents<T: 'static + Document>(
+    pub async fn get_documents<T: 'static + DeserializeOwned + Serialize + std::fmt::Debug>(
```

Comment:
ditto

Reply:
We can also remove the `std::fmt::Debug` here from what I see :+1: